apps/gro/controllers/confirm.js

```diff
@@ -27,32 +27,18 @@ ConfirmController.prototype.saveValues = function saveValues(req, res, callback)
 
     var subject = locali18n.translate('pages.email.information.subject');
 
-    var d = {
-      values: (function getSessionValues(data) {
-        var r = {};
-        for (var prop in data) {
-          r[prop] = data[prop];
-        }
-        return r;
-      }(req.sessionModel.attributes))
+    var data = {
+      values: Object.assign({}, req.sessionModel.attributes)
     };
 
     var dateTime = new Date();
-    d.values.reportDate = dateTime.toISOString();
-    d.email = d.values['email-text'];
+    data.values.reportDate = dateTime.toISOString();
+    data.email = data.values['email-text'];
```

Comment:
actually a more concise way of doing this would be in the data assignment:
```js
var data = {
  values: Object.assign({}, req.sessionModel.toJSON(), {
    reportDate: (new Date()).toISOString()
  }),
  email: req.sessionModel.get('email-text')
};
```